drizzlepac/hlautils/product.py

```diff
@@ -265,7 +265,7 @@ def align_to_gaia(self, catalog_name='GAIADR2', headerlet_filenames=None, output
                     align_table.perform_fit(method_name, catalog_name, ref_catalog)
                     align_table.select_fit(catalog_name, method_name)
                     align_table.apply_fit(headerlet_filenames=headerlet_filenames,
-                                         fit_label=fit_label)
+                                         fit_label=fit_label, fitgeom=fitgeom)
```

Comment:
OK.  I guess I am crazy.  I guess the perform_fit() here does not need the fitgeom in this particular invocation (?), but I do not see that apply_fit() even accepts a fitgeom parameter, and it does not do anything with the value.  Do I just not have the Python voodoo?

Reply:
No, you were not going crazy.  My edits happened over 2 days and I overlooked the mismatch on where this parameter was getting set.  This has been corrected in e78adcc.